jujugui/static/gui/src/app/components/machine-view/machine-view.js

```diff
@@ -399,6 +404,147 @@ YUI.add('machine-view', function() {
         `${unitCount} unit${unitPlural}`;
     },
 
+    /**
+      Sort the machines.
+
+      @method _sortMachines
+      @param {Array} machines A list of machines.
+      @param {String} method The sort method.
+      @returns {Array} A sorted list of machines
+    */
+    _sortMachines: function(machines, method) {
+      var sortMethod = this._getSortMethod(method);
+      return machines.sort(function (a, b) {
+        var sortedA = sortMethod(a);
+        var sortedB = sortMethod(b);
+        // Phantomjs only likes sort to return integer values so we have to
```

Comment:
I think this is the case in all browsers, it may just work by fluke in the browser.